OneCommunityGlobal/HighestGoodNetworkApp
# Papia create suggestion icon

# Description
(PRIORITY HIGH) Jae: Add “Suggestions” Icon to Dashboard icons section
It should function just like the Report App Bug icon (see below) but have the option to be submitted anonymously. 
Should have a dropdown that includes these options:
Suggestion Category: 
To identify and remedy poor client and/or user service experiences
To identify bright spots and enhance positive service experiences
To make fundamental changes to our programs and/or operations
To inform the development of new programs/projects
To identify where we are less inclusive or equitable across demographic groups
To strengthen relationships with the people we serve
To understand people's needs and how we can help them achieve their goals
Other
Would you like a followup/reply regarding this feedback? 
Should include an edit button for an Owner to add to the above “Suggestion Category:” dropdown
Should include an edit button for an Owner to add an additional information collection field if needed

## Related PRS (if any):
This frontend PR is related to the #440 backend PR.

## Main changes explained:
Added a form to take user suggestion, with one suggestion dropdown and radio field for feedback confirmation. For owner class added two edit buttons to add new suggestion category to the dropdown option, and adding extra text field. 

## How to test:
1. please follow backend PR 440 first to test this PR
2. check into current branch frontend and backend
3. do `npm run start:local` to run this PR locally
4. log as owner
5. go to dashboard→ click suggestion icon, as a owner try adding new suggestion category and new field, refresh the page to check if changes stays. 
6. fill up the required fields and submit, if you have configured the backend for sending email to your gmail address correctly, you will receive a suggestion email.

## Screenshots or videos of changes:
#Adding field and submitting form:
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/86232710/73595da8-f551-4a79-a849-c531456c66fd

#Received suggestion email:
![suggestionemail](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/86232710/35072478-26a6-40fb-b0db-05932867d988)
